src/main/java/com/cisco/mongodb/aggregate/support/query/AggregateQueryProvider.java

```diff
@@ -112,6 +112,11 @@ public boolean isAllowDiskUse() {
     return aggregateAnnotation.isAllowDiskUse();
   }
 
+  @Override
+  public long getMaxTimeMS() {
+    return aggregateAnnotation.maxTimeMS();
+  }
```

Comment:
We can move this to the base class and remove the overrides.  The default value of the annotation is already DEFAULT_MAX_TIME_MS so it'll take care of that.

Reply:
My first pass at this didn't allow the value to be overridden in annotation - this is left over from that. Will remove now.